.gitignore

```diff
@@ -2,3 +2,4 @@
 /composer.lock
 /phpunit.xml
 /vendor
+/.idea
```

Comment:
This should live in your global gitignore, not in this project 

For example, I have [this one](https://github.com/lyrixx/dotfiles/blob/master/.gitglobalexclude), included with [`.gitconfig`](https://github.com/lyrixx/dotfiles/blob/master/.gitconfig#L42)